redux/actions/adminRequests.js

```diff
@@ -0,0 +1,19 @@
+import http from "../../src/utils/axios";
+import { GET_REQUESTS, GET_REQUESTS_ERROR} from "./types";
+
+export const adminRequests = () => async(dispatch) => {
+  try {
+    const res = await http.get(`/api/request/manager`);
+    console.log("The response is:",res.data)
```

Comment:
You might need to remove this _console.log_ if it was used for debugging purposes